src/python/library/setup.py

```diff
@@ -76,8 +76,10 @@ def req_file(filename, folder="requirements"):
 extras_require["all"] = list(chain(extras_require.values()))
 
 platform_package_data = []
-if PLATFORM_FLAG != "any":
+if "linux" in PLATFORM_FLAG:
     platform_package_data += ["libcshm.so"]
+elif PLATFORM_FLAG == "win_amd64":
+    platform_package_data += ["cshm.dll"]
```

Comment:
Should we add an else block catching unsupported platform flags?

Reply:
The platform flag is currently passed in from `build_wheel.py`, so, from our perspective, the possible platforms are all handled. If the platform does not contain the substring "linux" or is not "win_amd64", it will be "any". In this case, we do not include either shared library object.

Are you suggesting raising an error in the event the `setup.py` script is called independently? Like so:
```python
if "linux" in PLATFORM_FLAG:
    platform_package_data += ["libcshm.so"]
elif PLATFORM_FLAG == "win_amd64":
    platform_package_data += ["cshm.dll"]
elif PLATFORM_FLAG != "any":
    raise Exceotion(f"Unsupported platform flag \"{PLATFORM_FLAG}\" specified.") 
```